### ISSUE TITLE
Fix whatever is wrong with twisted.trial.util's module docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5970]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5970|
|Type|enhancement|
|Created|2012-09-12 18:25:59Z|
|Branch|https://github.com/twisted/twisted/tree/trial-util-docstring-5970|

pydoctor complains that it is not valid epytext, but I can't see why.

<details><summary>Searchable metadata</summary>

```
trac-id__5970 5970
type__enhancement enhancement
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__branches_trial_util_docstring_5970 branches/trial-util-docstring-5970
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__documentation documentation
time__1347474359000000 1347474359000000
changetime__1347538301000000 1347538301000000
version__None None
owner__exarkun exarkun
cc__jml
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [35616]) Branching to 'trial-util-docstring-5970'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [35617]) Perhaps vars need to go at the end

refs [#5970](https://github.com/twisted/twisted/issues/5970)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

That seemed to do it.  [Build results](http://buildbot.twistedmatrix.com/builders/documentation/builds/2477)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @exarkun|
|-|-|

Thanks, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [35621]) Merge trial-util-docstring-5970

Author: exarkun
Reviewer: thijs
Fixes: [#5970](https://github.com/twisted/twisted/issues/5970)

Correct epytext error in `twisted.trial.util` docstring.
